### ISSUE TITLE
misdetects spdx-license-ids LICENSE

### ISSUE THREAD

**Issue by graingert** [davglass/license-checker#77]
it comes out as

```
[
"Glide",
"Abstyles",
"AFL-1.1",
"AFL-1.2",
"AFL-2.0",
"AFL-2.1",
"AFL-3.0",
"AMPAS",
"APL-1.0",
"Adobe-Glyph",
"APAFML",
"Adobe-2006",
"AGPL-1.0",
"Afmparse",
"Aladdin",
"ADSL",
"AMDPLPA",
"ANTLR-PD",
"Apache-1.0",
"Apache-1.1",
"Apache-2.0",
...
```

See https://procensus.com/acknowledgements.html#spdx-license-ids

**Comment by davglass** [comment]
My apologies, but I don't grok what you are reporting. Can you add some more details please?

**Comment by graingert** [comment]
I'm expecting the contents of https://github.com/shinnn/spdx-license-ids/blob/master/LICENSE but I actually get https://github.com/shinnn/spdx-license-ids/blob/master/spdx-license-ids.json

for the spdx-license-ids library

**Comment by davglass** [comment]
Ok, I got it now. The `licenseFile` in the parsed data is picking up the JSON file and not the license file. 

I have code that tries to look up the license by name in the package. And it's too broad. Let me see if I can make it a little more strict.

**Comment by davglass** [comment]
Published in `license-checker@7.1.1`
